java/flight/flight-jdbc-driver/pom.xml

```diff
@@ -143,6 +143,12 @@
             <artifactId>bcpkix-jdk15on</artifactId>
             <version>1.61</version>
         </dependency>
+
+        <dependency>
+            <groupId>joda-time</groupId>
```

Comment:
Is JodaTime really necessary? I thought most of its functionality is part of JDK 8.

Reply:
The joda-time lib helps simplify parsing java.time.Duration and java.time.Period in fewer methods. It's also what is used in Dremio, so it's easier to ensure it has the same value.